ui/src/app/constants/texts.constants.ts

```diff
@@ -51,6 +51,12 @@ export const texts = {
   RUN_WORKFLOWS_SUCCESS_NOTIFICATION: 'Workflows have been submitted.',
   RUN_WORKFLOWS_FAILURE_NOTIFICATION: "Sorry, workflows couldn't be submitted. Please try again.",
 
+  KILL_JOB_CONFIRMATION_TITLE: 'Kill running job',
+  KILL_JOB_CONFIRMATION_CONTENT:
+    'Are you sure you want to kill running job? The operation cannot be reverted and operation may not take effect.',
+  KILL_JOB_SUCCESS_NOTIFICATION: 'Kill job request passed, but it does not mean that job is killed. Please refresh page and verify status.',
```

Comment:
I'd rephrase "Request to kill job passed" Otherwise it is a bit confusing it could mean killing a job request (whatever a job request is) or requesting to kill the job. Again it is 100% correct, but a bit confusing. How about

"Request to kill the job passed. Please refresh the page after a few seconds and verify its status"

Reply:
Done